tibber/__init__.py

```diff
@@ -269,7 +307,9 @@ async def fetch_consumption_data(self):
             ) < now - dt.timedelta(hours=n_hours + 24):
                 self.hourly_consumption_data = []
             else:
-                n_hours = (now - self.last_cons_data_timestamp).total_seconds() / 3600
+                n_hours = int(
+                    (now - self.last_cons_data_timestamp).total_seconds() / 3600 # type: ignore[operator]
```

Comment:
This is the line where I ignore a mypy error. `self.last_cons_data_timestamp` will always be defined at this point, right?